suite-native/module-home/src/components/PortfolioGraphHeader.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+
+import { format } from 'date-fns';
+import { atom, useAtom } from 'jotai';
+
+import { useFormatters } from '@suite-common/formatters';
+import { Box, Text } from '@suite-native/atoms';
+import { Icon, IconName } from '@trezor/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { ExtendedGraphPoint, LineGraphPoint } from '@suite-common/wallet-graph';
+
+// use atomic jotai structure for absolute minimum re-renders and maximum performance
+// otherwise graph will be freezing on slower device while point swipe gesture
+const selectedPointAtom = atom<ExtendedGraphPoint>({
+    value: 0,
+    date: new Date(),
+    originalDate: new Date(),
+});
+// reference is usually first point, same as Revolut does in their app
+const referencePointAtom = atom<ExtendedGraphPoint>({
+    value: 0,
+    date: new Date(),
+    originalDate: new Date(),
+});
+
+export const writeOnlySelectedPointAtom = atom<null, ExtendedGraphPoint | LineGraphPoint>(
+    null, // it's a convention to pass `null` for the first argument
+    (_get, set, updatedPoint) => {
+        // LineGraphPoint should never happen, but we need it to satisfy typescript because of originalDate
+        set(selectedPointAtom, updatedPoint as ExtendedGraphPoint);
+    },
+);
+export const writeOnlyReferencePointAtom = atom<null, ExtendedGraphPoint>(
+    null,
+    (_get, set, updatedPoint) => {
+        set(referencePointAtom, updatedPoint);
+    },
+);
+
+const percentageDiff = (a: number, b: number) => {
```

Comment:
since this is not global util function I would prefer not using a / b as parameter names (at least somethingA and somethingB to improve readability?)

Reply:
This is just math formula so I think using just letters is OK because it's normal in math. It should not represent any specific value, only generic numbers.

Reply:
I can name it number1/2 but I do not think it will be better 😅